### DISCUSSION TITLE
Another exception was thrown: Exception: Please provide ShowCaseView context

### DISCUSSION THREAD

**Discussion by xitsumit** [SimformSolutionsPvtLtd/flutter_showcaseview#281] [Q&A] [answered]
Another exception was thrown: Exception: Please provide ShowCaseView context

**Comment by Sahil-Simform** [comment, marked as answer]
Please ensure that you use the BuildContext provided in the builder of ShowCaseWidget or one of its child widgets to access ShowCaseView, as it relies on the InheritedWidget mechanism.

```dart
ShowCaseWidget(
  builder : (context) => Somewidget(),
),
```
